src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GrouppedByTypeTreeViewProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Snapshot;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies
+{
+    [Export(typeof(IDependenciesTreeViewProvider))]
+    [AppliesTo(ProjectCapability.DependenciesTree)]
+    [Order(Order)]
+    internal class GrouppedByTypeTreeViewProvider : TreeViewProviderBase
```

Comment:
i was going to implement it too and had a right click command to swicth views, but i will cut it for now. Will remove this provider